src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid $white-border-color;
+  border-radius: $card-border-radius;
+  width: $card-width;
+
+  &__info-logo {
+    width: 160px;
+    height: 134px;
+    padding: 0 3px;
+    margin-bottom: 40px;
+  }
+
+  &__info-title {
+    color: $dark-color;
+    font-weight: 500;
+    font-size: 12px;
+
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__info-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey-color;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    @extend %flex-between-center;
+
+    line-height: 14px;
+    margin-bottom: 24px;
+
+    &-mark {
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    @extend %flex-between-center;
+    margin-bottom: 16px;
+
+    &-text {
+      font-size: 12px;
+      line-height: $card-price-line;
+      color: $grey-color;
+    }
+
+    &-amount {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: $card-price-line;
+      color: $dark-color;
+    }
+  }
+
+  &__button {
+    height: 38px;
+
+    color: $white-color;
+    background: $sea-color;
+
+    @extend %flex-center-center;
+    text-decoration: none;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    border: 1px solid $sea-color;
+    border-radius: $card-border-radius;
+
+    transition: color 0.5s ease-out, background-color 0.5s ease-out;
```

Comment:
Would be nice to create a variable for duration, then we can use it for every transition to make it's duration consistent with other page transitions